LocalSettings.php

```diff
@@ -120,9 +121,10 @@
             'auth' => false,
             'username' => 'noreply',
             'password' => $wmgSMTPPassword,
-        ),
-    ),
```

Comment:
Both of these removals need to be removed. the SMTP arrays above need to be closed